src/hist.jl

```diff
@@ -38,7 +104,17 @@ Plots.@deps cdensity path
 # ---------------------------------------------------------------------------
 # StatsBase.Histogram
 
-@recipe function f(h::StatsBase.Histogram)
-    seriestype := :histogram
-    h.edges[1], h.weights
+@recipe function f{T, E}(h::Histogram{T, 1, E})
+    seriestype := :binned
+    (h.edges[1], h.weights)
+end
+
+@recipe function f{T, E}(h::Histogram{T, 2, E})
+    x := _hist_edge_centers(h.edges[1])
+    y := _hist_edge_centers(h.edges[2])
+    z := Surface(h.weights)
+    match_dimensions := true
+    linewidth := 0
```

Comment:
This should probably be `linewidth --> 0`, no reason to constrain the user here.